Backpack/src/main/res/layout/view_bpk_skeleton_circle.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
same here - I think this can be a `FrameLayout`?

Reply:
ok